### PR TITLE
Minor: CSS combinators

### PR DESCRIPTION
**Pull request by estelle**
improve the legibility of the combinator section to help readers more quickly move on to the correct page.

### CODE REVIEW

File: files/en-us/web/css/css_selectors/selectors_and_combinators/index.md [hunk 1]
```diff
@@ -99,10 +111,10 @@ h2 + p + p {
   Nothing selectors match; not even the last one.
 </h2>
 <p>The first paragraph is underlined. All the paragraphs are 1.1rem</p>
-<p>The second paragraph is red. This and the next paragraphs are italic</p>
+<p>The second paragraph is red. This and the following paragraphs are italic</p>
```
Comment (dipikabh) [inline files/en-us/web/css/css_selectors/selectors_and_combinators/index.md:114]:
```suggestion
<p>The second paragraph is red. This and the following paragraphs are italic.</p>
```

File: files/en-us/web/css/css_selectors/selectors_and_combinators/index.md [hunk 2]
```diff
@@ -99,10 +111,10 @@ h2 + p + p {
...
 <p>The third paragraph is NOT red. It is italic and 1.1rem.</p>
 <p class="myClass">
-  Monospace font as this is a child of #myId. It is also italic and 1.1rem.
+  Monospace font as this both is myClass and is a child of #myId. It is also italic and 1.1rem.
```
Comment (dipikabh) [inline files/en-us/web/css/css_selectors/selectors_and_combinators/index.md:117]:
```suggestion
  This is in Monospace font as this is .myClass as well as a child of #myId. It is also italic and 1.1rem.
```
The font is not really showing any styling different from the rest of the text in this example (I'm using Firefox)

**Review by dipikabh — APPROVED** [review]
Thanks, @estelle! The H3 additions look good. I've added a suggestion for the example text.

There seems to be a markdownlint error somewhere - possibly that's why no preview page was generated 🤔

File: files/en-us/web/css/css_selectors/selectors_and_combinators/index.md [hunk 3]
```diff
@@ -99,10 +111,11 @@ h2 + p + p {
   Nothing selectors match; not even the last one.
 </h2>
 <p>The first paragraph is underlined. All the paragraphs are 1.1rem</p>
-<p>The second paragraph is red. This and the next paragraphs are italic</p>
+<p>The second paragraph is red. This and the following paragraphs are italic.</p>
 <p>The third paragraph is NOT red. It is italic and 1.1rem.</p>
 <p class="myClass">
-  Monospace font as this is a child of #myId. It is also italic and 1.1rem.
+  Monospace font as this both is myClass and is a child of #myId. It is also
```
Comment (estelle) [inline files/en-us/web/css/css_selectors/selectors_and_combinators/index.md:117]:
```suggestion
  Underlined, as this both is myClass and is a child of #myId. It is also
```

File: files/en-us/web/css/css_selectors/selectors_and_combinators/index.md [hunk 4]
```diff
@@ -85,7 +97,7 @@ h2 + p + p {
   text-decoration: underline;
 }
 #myId > .myClass {
-  font-family: monospace;
+  outline: 3px dashed #red;
```
Comment (estelle) [inline files/en-us/web/css/css_selectors/selectors_and_combinators/index.md:100]:
```suggestion
  outline: 3px dashed red;
```

**Pull request merged**